modules/installation-user-infra-exporting-common-variables.adoc

```diff
@@ -54,6 +67,20 @@ $ export REGION=`jq -r .gcp.region <installation_directory>/metadata.json`
 ----
 <1> For `<installation_directory>`, specify the path to the directory that you stored the installation files in.
 
+ifdef::shared-vpc[]
```

Comment:
Could you please add a note to explain that base_domain is the public dns zone on host project?